russian/kluch-na-14/kluch.md

```diff
@@ -0,0 +1,3 @@
+Неопрятный мужичок с жилеткой поверх пуховика подбежал откуда то сзади и застрял между двух деревьев, пытаясь пропихнуть свой несоизмеримо круглый живот между двух березок. В какой-то момент он понял, что самому ему не выбраться и посмотрел на вас взглядом хищного кота, попавшего в крысиную засаду. Как будто это вы были в этом виноваты. Сразу стало ясно - он русский. Даже в этом взгляде можно было прочитать мольбу о помощи. Ваши действия:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.